data/json/mapgen/homeimprovement_superstore.json

```diff
@@ -0,0 +1,291 @@
+[
+    {
+        "type"  : "item_group",
+        "id"    : "pwr_clothing",
+        "items" : [
+            ["hat_hard",         100],
+            ["boots",            35],
+            ["glasses_safety",   50]
+        ]
+    },
+        {
+        "type"  : "item_group",
+        "id"    : "pwr_tools",
+        "items" : [
+            ["flashlight",       35],
+            ["wrench",           50],
+            ["screwdriver",      35]
+        ]
+    },
+        {
+        "type"  : "item_group",
+        "id"    : "pwr_parts",
+        "items" : [
+            ["cable",            65],
+            ["power_supply",     50],
+            ["amplifier",        50]
+        ]
+    },
+	    {
+        "type"  : "item_group",
+        "id"    : "wood",
+        "items" : [
+            ["2x4",            80],
+            ["power_supply",     50],
+            ["amplifier",        50]
+        ]
+    },
+    {
+        "type" : "mapgen",
+        "om_terrain" : ["hdwr_large_entrance"],
+        "method": "json",
+        "weight": 100,
+        "object": {
+            "rows":[
+            ".|s....................|",
+            ".|t.............rrrrrrr|",
+            ".......................|",
+            ".................ccc...|",
+            ".---.....r|r...........|",
+            ".|CT.....r|r.....ccc...|", 
```

Comment:
Hm, I think I'm missing something. I've assigned terrain for my furniture and replaced the t_door_clear_c and a vending machine that also didn't exist and it's still giving me this error. I've checked all of my terrain with terrain.json, and they're all there.